CS483/CS483/Kartaclysm/Components/ComponentRacer.h

```diff
@@ -36,9 +36,9 @@ namespace Kartaclysm
 		virtual void Update(const float p_fDelta) override;
 
 		HeatStroke::GameObject* GetKart() {return m_pKart;}
-		void SetKart(HeatStroke::GameObject* p_pKart);
+		void SetKart(HeatStroke::GameObject* p_pKart) { m_pKart = p_pKart; }
```

Comment:
Is there a reason we don't want the kart and driver to be children of the racer any more?